integration_tests/e2e/overview/supportNeedsTab.cy.ts

```diff
@@ -14,11 +14,12 @@ context('Prisoner Overview page - Support Needs tab', () => {
     cy.task('stubLearnerEducation')
     cy.task('stubGetInduction')
     cy.task('getActionPlan')
+    cy.task('stubGetAllPrisons')
   })
 
   it('should display Support Needs data', () => {
     // Given
-    cy.task('stubLearnerProfile')
+    cy.task('stubLearnerAssessments')
```

Comment:
Minor changes to the existing Support Needs page cypress test to use the new stubs for the Curious V2 /learnerAssessments endpoints, because that is what the implementation now calls 👍 
The tests themselves have not changed 👍